### ISSUE TITLE
Obtaining MimeContent for an mail item

### ISSUE THREAD

**Issue by nickalbrecht** [microsoftgraph/microsoft-graph-docs#9286]
Example 4 on the page states that you can get the MIME content of an item using the SDK in C# by using `.Content`, but actually trying to use that code gives a compile error stating that

> `IMessageRequestBuilder` does not contain a definition for `Content`, and no extension method `Content` accepting a first argument of type `IMessageRequestBuilder` could be found

I'm not sure if I'm missing a different package to include or something else, but for the moment I'm going off of the assumption that the documentation might be wrong, strictly because it's the only example on this page that has a property hanging off the Messages collection before the call to `Request()`? I've done some digging through the source on github, but I can't seem to wade through it in a meaningful way to see if there is a `.Content` extension method elsewhere in the SDK

Package used: Microsoft.Graph 3.8.0

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d0a4123-11fb-5985-3e94-eb0834e93369
* Version Independent ID: b1556931-c2fd-7400-4e45-d666a8a53540
* Content: [Get message - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/message-get?view=graph-rest-beta&tabs=csharp)
* Content Source: [api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/message-get.md)
* Product: **outlook**
* Technology: **microsoft-graph**
* GitHub Login: @svpsiva
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by nickalbrecht** [comment]
I didn't notice at first, but this seems to be an issue others have had too #840  I ended up downloading the entire Graph SDK for dotnet trying to find this and I don't see it in the code at all. (try searching for all instances of "$value", since that is supposed to be appended to the end of the URL to get the mime content). In the end I ended up mimicking the code style of the SDK to add the missing functionality. Hope this helps anyone else looking for this until it gets fixed.

This is an example of how to use this code following the style from Example 4. Specifically note the parenthesis on the `Content()` due to it being an extension method now

```csharp
GraphServiceClient graphClient = new GraphServiceClient(authProvider);

var stream = await graphClient.Users["user@example.com"].Messages["4aade2547798441eab5188a7a2436bc1"]
	.Content()
	.Request()
	.GetAsync();
```


```csharp
public static class MessageContentRequestBuilderExtensionMethods
{
	public static IMessageContentRequestBuilder Content(this Microsoft.Graph.IMessageRequestBuilder messageRequestBuilder)
	{
		return new MessageContentRequestBuilder(messageRequestBuilder.AppendSegmentToRequestUrl("$value"), messageRequestBuilder.Client);
	}
}

public interface IMessageContentRequestBuilder
{
	IMessageContentRequest Request(IEnumerable<Microsoft.Graph.Option> options = null);
}
public class MessageContentRequestBuilder : Microsoft.Graph.BaseRequestBuilder, IMessageContentRequestBuilder
{
	public MessageContentRequestBuilder(string requestUrl, Microsoft.Graph.IBaseClient client)
		: base(requestUrl, client)
	{
	}

	public IMessageContentRequest Request(IEnumerable<Microsoft.Graph.Option> options = null)
	{
		return new MessageContentRequest(this.RequestUrl, this.Client, options);
	}
}

public interface IMessageContentRequest
{
	/// <summary>
	/// Gets the stream.
	/// </summary>
	/// <returns>The stream.</returns>
	System.Threading.Tasks.Task<Stream> GetAsync();

	/// <summary>
	/// Gets the stream.
	/// </summary>
	/// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
	/// <param name="completionOption">The <see cref="HttpCompletionOption"/> to pass to the <see cref="IHttpProvider"/> on send.</param>
	/// <returns>The stream.</returns>
	System.Threading.Tasks.Task<Stream> GetAsync(CancellationToken cancellationToken, System.Net.Http.HttpCompletionOption completionOption = System.Net.Http.HttpCompletionOption.ResponseContentRead);
}
public class MessageContentRequest : Microsoft.Graph.BaseRequest, IMessageContentRequest
{
	/// <summary>
	/// Constructs a new FileAttachmentContentRequest.
	/// <param name="requestUrl">The request URL.</param>
	/// <param name="client">The <see cref="IBaseClient"/> for handling requests.</param>
	/// <param name="options">Query option name value pairs for the request.</param>
	/// </summary>
	public MessageContentRequest(
		string requestUrl,
		Microsoft.Graph.IBaseClient client,
		IEnumerable<Microsoft.Graph.Option> options)
		: base(requestUrl, client, options)
	{
	}

	/// <summary>
	/// Gets the stream.
	/// </summary>
	/// <returns>The stream.</returns>
	public System.Threading.Tasks.Task<Stream> GetAsync()
	{
		return this.GetAsync(CancellationToken.None);
	}

	/// <summary>
	/// Gets the stream.
	/// </summary>
	/// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
	/// <param name="completionOption">The <see cref="HttpCompletionOption"/> to pass to the <see cref="IHttpProvider"/> on send.</param>
	/// <returns>The stream.</returns>
	public System.Threading.Tasks.Task<Stream> GetAsync(CancellationToken cancellationToken, System.Net.Http.HttpCompletionOption completionOption = System.Net.Http.HttpCompletionOption.ResponseContentRead)
	{
		this.Method = "GET";
		return this.SendStreamRequestAsync(null, cancellationToken, completionOption);
	}
}
```

**Comment by nickalbrecht** [comment]
Was updating some package references today and noticed I was still using 3.35.0. Updated to the most recent version of 4.x on Nuget and it looks like this issue was resolved. Some surface digging seems to indicate it was fixed in the early 4.0.0 betas.
